README.rst

```diff
@@ -25,15 +25,32 @@ Requirements
 * wxPython (http://www.wxpython.org) >= 2.8
   (wxPython Phoenix, i.e. release >=4.0.0, is supported)
 
-If you have a choice, you should prefer wxPython Phoenix running on Python 3.x.
+If you have a choice, you should prefer wxPython Phoenix running on Python 3.x. *the Phoenix version of wxPython
+can be pip installed with `python -mpip install wxPython`.*
```

Comment:
This does not really work for Linux.
I will place a link to https://wxpython.org/pages/downloads/ , though.

Reply:
For Python 3 & Phoenix pip should work find on Linux *(but may require sudo depending on installation)*

Reply:
It will only work if you have the required libraries installed as it will need to build from source. E.g. I just tried on my Ubuntu 19.10 VM.
The recommended way is to use the wheels or the distribution packages.